rs/nns/governance/canister/canister.rs

```diff
@@ -161,6 +161,7 @@ fn set_governance(gov: Governance) {
 fn schedule_timers() {
     schedule_seeding(Duration::from_nanos(0));
     schedule_adjust_neurons_storage(Duration::from_nanos(0), NeuronIdProto { id: 0 });
+    schedule_spawn_neurons();
```

Comment:
Approving, but let's check in the morning with Zurich teammates to see if any objections.

Reply:
Looks like there are no objections. Resolving.